cmd/ctr/commands/images/import.go

```diff
@@ -146,33 +145,26 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 				opts = append(opts, image.WithNamedPrefix(prefix, overwrite))
 			}
 
-			var platSpec ocispec.Platform
-			// Only when all-platforms not specified, we will check platform value
-			// Implicitly if the platforms is empty, it means all-platforms
+			// Even with --all-platforms, only the default platform layers are unpacked,
+			// for compatibility with --local.
+			//
+			// This is still not fully compatible with --local, which only unpacks
+			// the strict-default platform layers.
```

Comment:
(The transfer API should support specifying a strict platform matcher, but it is beyond the scope of this PR)